### PR TITLE
controllers specs-PullRequest

### PR DESCRIPTION
**Pull request by IvonneBenitesRodriguez**
**About this project**<br/>
This part of the project is about setting up tests on the controllers that we added to our Blog app.

**For it:**<br/>
- A featured branch named controllers-specs is created.
- RSpec Gem was added to the project inside our Gemfile.
- We created a folder named requests inside our spec folder to store our spec files for controllers.
- Linters were passed.
- We have respected Gitflow.

**Goal**<br/>
The goal is to implement tests for controllers and feel comfortable about it.

### CODE REVIEW

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    before { get users_path }
+
+    it 'responds successfully' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'displays a list of users' do
+      expect(response.body).to include('<h1>A list of users</h1>')
+    end
+  end
```
Comment (Gambit142) [inline spec/requests/users_spec.rb]: - [x] When writing the _**request spec tests**_ for the `index` action of the `Users` controller, you forgot to include a test case that ensures that the correct template is being rendered when requests are made to this action. Kindly include this test. 👍 😎

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @IvonneBenitesRodriguez 👋, 

Good job so far!

### Highlights
- Setup linters correctly ✔️ 
- Use of `Rspec` testing Library ✔️ 
- Wrote request specs for `Users` and `Posts` controllers ✔️ 
- All tests pass ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: spec/requests/users_spec.rb (hunk 1 above)
Comment (IvonneBenitesRodriguez) [reply spec/requests/users_spec.rb]: Dear @Gambit142 , 👋🏾
Very nice to meet you, 
thank you so much for your kind suggestions, 
I appreciate it, I modified the part you suggested, 
Best regards,

Comment (Whoistolu) [reply spec/requests/users_spec.rb]: - Well resolved 👍🏽 @IvonneBenitesRodriguez

**Review by Whoistolu — APPROVED** [review]
### Hi @IvonneBenitesRodriguez 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**